test/tests/executioners/eigen_convergence_issue/tests

```diff
@@ -0,0 +1,29 @@
+[Tests]
+  [testb]
+    type = RunApp
+    input = 'b.i'
+    expect_out = '4 Nonlinear'
+    absent_out = '5 Nonlinear'
+  []
+  [test11]
+    type = RunApp
+    input = 'a.i'
+    expect_out = '11 Nonlinear'
+    absent_out = '12 Nonlinear'
+    cli_args = '-eps_tol 1e-10'
+  []
+  [test7]
+    type = RunApp
+    input = 'a.i'
+    expect_out = '7 Nonlinear'
+    absent_out = '8 Nonlinear'
+    cli_args = 'Problem/bx_norm=fluxintegral -eps_tol 1e-10'
+  []
+  [test4]
+    type = RunApp
+    input = 'a.i'
+    expect_out = '4 Nonlinear'
+    absent_out = '5 Nonlinear'
+    cli_args = 'Problem/bx_norm=fluxintegral -eps_power_sign_normalization 0 -eps_tol 1e-10'
```

Comment:
What is this `-eps_power_sign_normalization`?

Reply:
Whether to multiply the Bx vector by the sign of the first non-zero element during normalization. This seems to ensure much better convergence for complex eigenvalues, but has poorer convergence for your particular test

Reply:
I actually do not quite like this sign of the first non-zero element approach. The dof in a solution vector could be negative although it is very likely the first dof is always positive.

Reply:
Well now you have the ability to disable it, and we can set that programmatically to be the default in MOOSE if you wish